app/views/layouts/application.html.erb

```diff
@@ -5,10 +5,10 @@
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
+    <%= stylesheet_link_tag "https://cdn.jsdelivr.net/npm/daisyui@4.6.0/dist/full.min.css" %>
```

Comment:
Within application.css, I want to override some of the classes that we get from this library. In order to do that, application.css needs to get included *after* this library is imported.

Reply:
I wondered if that was the case and started to type out a comment earlier today about it. That might be worth having in a source comment.